### ISSUE TITLE
Current issues

### ISSUE THREAD

**Issue by rodeknopje** [rodeknopje/epic-games-yoinker#4]
Epic added some extra protection when trying to login, ill fix this in the future. for now you have around 40% to login successfully.

**Comment by zadetek** [comment]
@rodeknopje Login doesn't work anymore

**Comment by rodeknopje** [comment]
> @rodeknopje Login doesn't work anymore

Yup seems like they added some extra protection :(, ill try to bypass it in the coming days.
